### PR TITLE
Add node_modules to handlebars partials locations

### PR DESCRIPTION
**Pull request by andygnewman**
cc: @i-like-robots 

Enable the finding of partials in node_modules

### CODE REVIEW

File: src/handlebars/index.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ module.exports = function (conf) {
 
 	return handlebars(app, {
 		partialsDir: [
-			directory + (options.viewsDirectory || '/views') + '/partials'
+			directory + (options.viewsDirectory || '/views') + '/partials',
+			directory + ('/node_modules/@financial-times')
```
Comment (i-like-robots) [inline src/handlebars/index.js]: I wonder if this should look in a subfolder to reduce cost at startup? Maybe `/views`? Might be good to have a convention, what do you think?

Comment (andygnewman) [inline src/handlebars/index.js]: If we made it more specific we'd have to do a glob to find the specific ones and change the referencing to be less specific.

With the specific reference in the template to the partial I don't think that this should be costly at start up as should only be invoked when looking for a specific partial.

File: src/handlebars/index.js [hunk 2]
```diff
@@ -13,7 +13,8 @@ module.exports = function (conf) {
 
 	return handlebars(app, {
 		partialsDir: [
-			directory + (options.viewsDirectory || '/views') + '/partials'
+			directory + (options.viewsDirectory || '/views') + '/partials',
+			directory + ('/node_modules')
```
Comment (debugwand) [inline src/handlebars/index.js]: again, limit to \@financial-times

**Comment by andygnewman** [comment]
Now scoped the node partials to `/@financial-times`

**Pull request merged**